examples/dynamics/controlled_integrator.ipynb

```diff
@@ -232,7 +233,8 @@
       "plt.plot(t, ref, 'k--', label='Exact')\n",
       "plt.plot(t, sim.data[A_probe][:,0], label='A (value)')\n",
       "plt.plot(t, sim.data[A_probe][:,1], label='A (control)')\n",
-      "plt.ylim([-1.1, 1.1])\n",
+      "plt.xlim(right=t[-1])\n",
```

Comment:
Hm, according to [this line in the master branch](https://github.com/matplotlib/matplotlib/blob/6b22bfc4d831c3f615cae87838127ec0e7aebd2d/lib/matplotlib/axes/_base.py#L2776) it's the other way around... since it's okay for `right < left`.